product_yahoo_auction_sst/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017-2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Products' Yahoo Auction information",
+    "version": "15.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Product",
+    "license": "AGPL-3",
+    "depends": ["product", "delivery", "hr"],
```

Comment:
```suggestion
    "depends": ["product", "delivery"],
```

Why is "hr" nessesary on depends.

Reply:
We use "staff_in_charge" field that is linked with "hr.employee". So, if we don't install hr module, we will get error.

Reply:
I think we should probably switch to reference res.users (in case they are supposed to have logins).  Not sure why it references hr.employee in 11.0.

@kanda999 Can you try and analyse.

Reply:
I will check now.